rates.md

```diff
@@ -36,23 +37,40 @@ module RATES
 
     syntax PotStep ::= "drip"
  // -------------------------
-    rule <k> Pot . drip => call Vat . suck VOW THIS ( CHI +Int (((#pow(DSR, TIME -Int RHO) *Int CHI) -Int CHI) ) ) ... </k>
+    rule <k> Pot . drip => call Vat . suck VOW THIS ( PIE *Rat CHI *Rat ( DSR ^Rat (TIME -Rat RHO) -Rat 1 ) ) ... </k>
          <this> THIS </this>
          <currentTime> TIME </currentTime>
-         <pot-chi> CHI => CHI +Int (((#pow(DSR, TIME -Int RHO) *Int CHI) -Int CHI) ) </pot-chi>
+         <pot-chi> CHI => CHI *Rat DSR ^Rat (TIME -Rat RHO) </pot-chi>
          <pot-rho> RHO => TIME </pot-rho>
          <pot-dsr> DSR </pot-dsr>
          <pot-vow> VOW </pot-vow>
-      requires TIME >=Int RHO
+         <pot-pie> PIE </pot-pie>
+      requires TIME >=Rat RHO
```

Comment:
We're strictly using Int for time values. I guess this is harmless but probably better to stick to Int for clarity.